dataflow/flex-templates/kafka_to_bigquery/.gcloudignore

```diff
@@ -0,0 +1,7 @@
+# Ignore everything from the Cloud Build command.
```

Comment:
A bit unexpected and incompatible with CloudBuild - you might wish to mention that in the README, but otherwise OK.